hw6_social.py

```diff
@@ -76,22 +76,26 @@ def parseState(fromString):
 Returns: list of strs
 '''
 def findHashtags(message):
-    words=message.replace('-', ' ').split()
-    hashTags=[]
-    for word in words:
-        if "#" in word:
+    hash = []
+    trigger = 0
+    result = '#'
+    for letter in message:
+        if letter == '#':
+            if trigger == 1:
+                hash.append(result)
+                result = '#'
+            trigger = 1
+            continue

```

Comment:
There is no need of continue in this code.

Reply:
issue resolved